src/graphql/device/TypeDefs.js

```diff
@@ -0,0 +1,91 @@
+const TypeDefs = [`
```

Comment:
Use these definitions, which were defined a while ago:

`type Device {
   id: String!
   label: String!
   attr: [Attr]
}
 
enum ValueType {
   NUMBER
   STRING
   BOOLEAN
   GEO
   UNDEFINED
}
 
type Attr {
   label: String!
   type: ValueType!
}
 
# A paginated list of Devices.
type DeviceListPage {
   # Identifies the total count of item.
   totalPages: Int!
   # Indicate the current Page.
   currentPage: Int!
   # List of Devices.
   devices: [Device]
}
 
input PageInput {
   # Identifies the page number.
   number: Int # default = 1
   # Identifies the page size.
   size: Int # default = 20
}
 
input FilterDeviceInput {
   # Return only devices that are named accordingly
   label: String # (prefix or suffix match)
}
 
input HistoryInput {
   devices: [HistoryDeviceInput]!
   beginTime: String # unix time,usar scalar DateTime?? será optional qdo tem lastN ?? 
   endTime: String # corrente caso vazio,usar scalar DateTime??
   lastN: Int # optional, default: ????
}
 
input HistoryDeviceInput{
   deviceID: String!
   attrs: [String]
}
 
type HistoryAttr {
   label: String!
   valueType: ValueType! #tipos da dojot
   value: String!
   timestamp: String! #unix time, usar scalar DateTime??
}
 
type History{
   deviceID: String!
   label: String!
   attrs: [HistoryAttr]
}
 
type Config {
   config: Object!
}
 
input ConfigInput {
   config: Object!
}
 
# será retornado APENAS os attrs dynamics
type Query {
   # irá ignorar qualquer atributo diferente de dynamic no retorno
   getDevices(page: PageInput filter: FilterDeviceInput): [DeviceListPage]
   # irá ignorar qualquer atributo diferente de dynamic no retorno
   getDeviceById(deviceId: String!): Device
   getHistory(filter: HistoryInput!): [History]
   getConfig(tenant: String!
               user: String!): Config
}
 
type Mutation {
   updateConfig(tenant: String!
                   user: String!
                   config: ConfigInput!): Config! # create or update if exist
}
`